src/test/java/com/treasuredata/client/TestServerFailures.java

```diff
@@ -18,7 +18,6 @@
  */
 package com.treasuredata.client;
 
-import com.google.common.base.Throwables;
 import com.google.common.io.CharStreams;
 import org.eclipse.jetty.http.HttpStatus;
 import org.eclipse.jetty.server.Request;
```

Comment:
https://gist.github.com/miniway/9a674feba93fd3b1bf5ee90b540a23fd

I think we can completely remove jetty dependency even in test cases